docs/quickstart.md

```diff
@@ -379,3 +379,25 @@ value to be more or less strict:
 ```python
 >>> httpx.get('https://github.com/', timeout=0.001)
 ```
+
+## Authentication
+
+HTTPX supports Basic and Digest HTTP authentication.
+
+To provide Basic authentication credentials you can provide a tuple of
+plaintext `str` or `bytes` objects as the `auth` argument to the request
+functions:
+
+```python
+>>> httpx.get("https://example.com", auth=("my_user", "password123"))
+```
+
+To provide credentials for Digest authentication you need to instantiate
+an `HTTPDigestAuth` object providing, again, the plaintext username and
+password as arguments. This object can be then passed as the `auth` argument
+to the request methods as above:
+
+```python
+>>> auth = httpx.DigestAuth("my_user", "password123")
```

Comment:
I think there's a bit of inconsistency here, right?
The class is currently `HTTPDigestAuth`.
I'd suggest that the class should be simply `DigestAuth`, as in this example, which scans more nicely to me.

Reply:
Requests names theirs `HTTPDigestAuth`.

Reply:
But we don't have an `auth` module anymore so it'll have to be renamed regardless. This is probably fine then. :) I do like the name without the HTTP prefix better.